### PR TITLE
add proto folder path to proto_path

### PR DESCRIPTION
**Pull request by erizocosmico**
This fixes the issue @smola had.

Depends on #81. Review just last commits.

### CODE REVIEW

File: example/server.proteus.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package example
 
 import (
 	"golang.org/x/net/context"
+	"gopkg.in/src-d/proteus.v1/example/categories"
```
Comment (smola) [inline example/server.proteus.go]: First internal, then external.

Comment (Serabe) [inline example/server.proteus.go]: This file is a generated file, is it really important for a generated file to follow that rule?

Comment (smola) [inline example/server.proteus.go]: :+1:

Comment (erizocosmico) [inline example/server.proteus.go]: We could gofmt all the files after generation, tho.

**Comment by Serabe** [comment]
Problem with Travis cloning the pr branch :$ Accepting the PR

**Pull request merged**